typescript/web/src/components/website/Hero/Hero.tsx

```diff
@@ -52,21 +53,12 @@ export function Hero() {
           align={{ lg: "center" }}
           justify="space-between"
         >
-          <Box flex="1" maxW={{ lg: "520px" }}>
-            {/* <Text
-              size="xs"
-              textTransform="uppercase"
-              fontWeight="semibold"
-              color={mode("brand.600", "brand.300")}
-              letterSpacing="wide"
-            >
-              Hire Talents
-            </Text> */}
+          <Box flex="1">
```

Comment:
You can use a `Flex` component directly instead ;)

Reply:
I think we had done this part together actually. I will look at it